### PR TITLE
Updated for USLCI mapping

### PR DESCRIPTION
**Pull request by ashleyedelen**
Addition of missing flow/context combinations and edits to the SecondaryContextMembership file to include chemical and groups resources and geological, biological and energy emissions.

### CODE REVIEW

**Comment by oftheheadland** [comment]
@ashleyedelen Hi Ashley. There's a mismatch with one of the input files: image

The 'Water' entry is unexpected and is causing errors with some of the tests for the input files. It seems like we're just missing the inclusion of Water in **BiologicalFlowables.csv** or possibly elsewhere.

image

**Comment by ashleyedelen** [comment]
This should be "Wood" not "Water" I fixed it.

**Comment by ashleyedelen** [comment]
The synonym was not put in "" marks and the script was reading the comma in the name as an additional column. I added "" to the synonym and this should fix the issue.

File: fedelemflowlist/input/ChemicalsFlowablePrimaryContexts.csv [hunk 1]
```diff
@@ -10465,8 +10466,9 @@ Ethylenediaminetetraacetic acid ammonium salt,Emission,Water
 Endrin aldehyde,Emission,Ground
 Endrin aldehyde,Emission,Water
 Lead stearate,Emission,Air
-Iron,Emission,Ground
+Iron,Ground
```
Comment (bl-young) [inline fedelemflowlist/input/ChemicalsFlowablePrimaryContexts.csv]: you are missing directionality here

File: fedelemflowlist/input/ChemicalsFlowables.csv [hunk 2]
```diff
@@ -4635,3 +4635,20 @@ cis-4-Methyl-2-pentene,691-38-3,C6H12 ,"2-pentene, 4-methyl-, (z)-;2-Pentene, 4
...
+Yttrium,7440-65-5,Y,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=36816,1
+Zirconium oxide,1314-23-4,ZrO2,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=42863,1
+Thulium,7440-30-4,Tm,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=36980,1
+Argon,7440-37-1,Ar,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=36974,1
+Ethyl cellulose,9004-57-3,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=37459,1
+Bis(N-Cyclohexyldiazeniumdioxy)copper,312600-89-8,C12H22CuN4O4,"Copper, bis[N-(hydroxy-.kappa.O)-N-(nitroso-.kappa.O)cyclohexanaminato]-",kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=76843,1
```
Comment (oftheheadland) [inline fedelemflowlist/input/ChemicalsFlowables.csv]: I'm still getting errors from the extra comma, it's confusing the parsing library. Could we use a semicolon instead? Are we doing this anywhere else where it works as expected?

**Comment by ashleyedelen** [comment]
Resolved the issue for Iron missing directionality

**Comment by ashleyedelen** [comment]
Edited Bis(N-Cyclohexyldiazeniumdioxy)copper. I removed the Chemical formula which had been accidentily added.

**Comment by ashleyedelen** [comment]
Okay I edited based on what Ben showed in the excel file the two rows that were not lining up.

**Comment by oftheheadland** [comment]
Those issues are resolved but some new ones: 

AssertionError: Items in the second set but not the first:
**'Benzo[b]fluoranthene'
'Asbestos'
'Disodium phosphonate'**

I'm looking into this but I believe these 3 are in **ChemicalsFlowablesPrimaryContexts.csv** but missing from **ChemicalsFlowables.csv**

**Comment by ashleyedelen** [comment]
Asbestsos needed to be moved to geological flows. Disodium phosphonate needed to have the name adjusted to Phosphonic acid, sodium salt and benzo[b]fluoranthene needed to have the brackets changed to parenthasis.

**Comment by oftheheadland** [comment]
@ashleyedelen 2 more, likely the same issue as the last 3?
 
image

**Comment by bl-young** [comment]
Bromate and bromide (emission, air) are included in geological flows primary context, but not in geological flows. I think these two records (lines 482/483) should be removed from geological flows primary contexts and added to chemical flows primary contexts (bromate is already there), assuming they are categorized correctly in flowables @ashleyedelen

**Comment by ashleyedelen** [comment]
Ben was correct, they just needed to be moved from geological to chemical flow context files and I double checked they are in the Chemical flowable file.

File: fedelemflowlist/input/GroupsFlowablePrimaryContexts.csv [hunk 3]
```diff
@@ -492,4 +492,16 @@ Tetrabromotrifluoropropanes,Emission,Air
 Tribromotetrafluoropropanes,Emission,Air
 Dibromopentafluoropropanes,Emission,Air
 Hexabromofluoropropanes,Emission,Air
-Propylbenzene isomers,Emission,Air
\ No newline at end of file
+Propylbenzene isomers,Emission,Air
+Adsorbable organic halides,Emission,Air
+"Hydrocarbons, chlorinated",Emission,Air
+"Hydrocarbons, halogenated
```
Comment (oftheheadland) [inline fedelemflowlist/input/GroupsFlowablePrimaryContexts.csv]: @ashleyedelen lines 498 and 499 have some issues, looks like 498 is missing some columns. Opening it in excel helps

**Pull request merged**